src/components/wizard/WizardStep.tsx

```diff
@@ -1,16 +1,23 @@
 import _ from 'lodash';
-import PropTypes from 'prop-types';
-import React from 'react';
+import React, {Component} from 'react';
 import {StyleSheet} from 'react-native';
 import View from '../view';
 import Text from '../text';
 import Image from '../image';
 import TouchableOpacity from '../touchableOpacity';
-import {PureBaseComponent} from '../../commons';
+import {asBaseComponent} from '../../commons/new';
 import Colors from '../../style/colors';
 import BorderRadiuses from '../../style/borderRadiuses';
 import Spacings from '../../style/spacings';
-import {States, StatesConfig} from './WizardStates';
+import {WizardStepProps, WizardProps} from './types';
+import {StatesConfig} from './WizardStates';
+
+// Includes private props from the Wizard
+interface Props extends WizardStepProps, Omit<WizardProps, 'onActiveIndexChanged | containerStyle'> {
+  index: number;
+  maxWidth: number;
+  onPress: WizardProps['onActiveIndexChanged'];
```

Comment:
They are all required?

Reply:
These are "private" props that are received from the `Wizard`, this was on-purpose.